src/newtondynamics_physics/SysNewton.cpp

```diff
@@ -328,8 +370,9 @@ void SysNewton::create_body(ActiveScene& rScene, ActiveEnt ent,
     // Make it easier to rotate
     NewtonBodySetAngularDamping(pBody, Vector3(1.0f, 1.0f, 1.0f).data());
 
-    // Set callback for updating position of entity and everything else
-    NewtonBodySetForceAndTorqueCallback(pBody, cb_force_torque);
+    // Set callback for applying force and setting transforms
+    NewtonBodySetForceAndTorqueCallback(pBody, &cb_force_torque);
+    NewtonBodySetTransformCallback(pBody, &cb_set_transform);
 
     // Set user data to entity
     // WARNING: this will cause issues if 64-bit entities are used on a
```

Comment:
in my opinion, we should explicitly trigger compiler errors if compiling for 32bit.

I don't think there is any reason to be supporting a 32bit platform in a post 2020 world. 64bit has been ubiquitous for basically a decade now.

Reply:
Basically something like

#if 32bit
#error "Get a better computer"
#endif

Reply:
Just realized a 32-bit computer wouldn't be able to support over 2^32 entities in the first place